### PR TITLE
UDP Command Application ASCII Mode

### PR DESCRIPTION
**Pull request by AarC10**
This closes #<Insert Issue Number Here>

# Description
Write a summary of what you changed. It is helpful to have context on why you made this change! 

# How Has This Been Tested?
Describe how you tested this change and any special setup configurations in case someone else wants to reproduce this.  

# Checklist:
- [ ] New functionality is documented in the necessary spots (i.e new functions documented in the header)
- [ ] Unit tests cover any new functionality or edge cases that the PR was meant to resolve (if applicable) 
- [ ] The CI checks are passing
- [ ] I reviewed my own code in the GitHub diff and am sure that each change is intentional

### CODE REVIEW

File: cmd/udp_command/udp_command.go [hunk 1]
```diff
@@ -130,38 +142,99 @@ func promptInput(scanner *bufio.Scanner, history [][]byte) ([]byte, error) {
...
+					result = append(result, 0)
+				default:
+					// If not a recognized escape, just add the character
+					result = append(result, input[i+1])
+				}
+				i++ // Skip the next character (the one after \)
+			} else {
+				// Regular character
+				result = append(result, input[i])
+			}
```
[the remark this answers is not in the document]
Comment (cowsed) [reply cmd/udp_command/udp_command.go]: kinda with the ai on this one, tho idk if i agree with the name
